// Copyright (c) 2017, Kai-Zhang
// Use of this source code is governed by a MIT license that can be
// found in the LICENSE file.
//
// Author: Kai Zhang (cs.zhangkai@outlook.com)

package orion.serialize;

enum NodeType {
    NODE_PERMANENT = 0;
    NODE_TEMP = 1;
}

message DataValue {
    optional string value = 1;
    optional NodeType type = 2;
    optional string owner = 3;
    optional int64 last_modified = 4;
}

